tests/README.md

```diff
@@ -50,6 +50,7 @@ The test environment prepares these before the tests are started.
 
 ## Description of tests
 
+TODO: translate this presentation into markdown format (.md)
```

Comment:
This looks pretty ugly when rendered. I also don't think we need it in the repository, the issue is sufficient.